node/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -208,7 +213,7 @@ getWalletSyncTip cWalId = preview (wsWalletInfos . ix cWalId . wiSyncTip)
 
 getWalletAddresses :: Query [CId Wal]
 getWalletAddresses =
-    map fst . sortOn (view wiCreationTime . snd) . HM.toList <$>
+    map fst . sortOn (view wiCreationTime . snd) . filter (view wiIsReady . snd) . HM.toList <$>
```

Comment:
I think that it would be nice to not only add this check here, but also throw an error with according message in `getWallet`, when given wallet isn't ready yet.

Reply:
Done!